vermouth/tests/data/integration_tests/tier-1/lysozyme_GO/martinize2/command

```diff
@@ -8,3 +8,4 @@ martinize2
 -go ../map.map 
 -noscfix
 -cys none
+-name molecule_0
```

Comment:
shouldn't this be 'molecule'? Also for the second  test

Reply:
No, this is what the test expects and I decided to be lazy about it. Before, the molecule name was defaulting to 'molecule_0', so the go_atomtypes get called molecule_0_{1...n}. Because the name now just defaults to 'molecule', giving it as 'molecule_0' here is what's needed to give the same output in the test

Reply:
Does this mean that if I have multiple proteins (=molecules) in my system, named `molecule_{0..n}` that I still end up with overlapping go atomtypes?

Reply:
No, because in a Go model the molecules are merged, so you just end up with a single molecule in the end. I guess in a system with multiple proteins that you've martinized with a go model you do, but with this you'd be able to avoid it by having the flexibility to give them different names

Reply:
Ok...
Maybe this is going a bit deep for this PR, but why the merge? Why not make a go model per protein? I realize the merge needs to be there *as an option*, but as default it feels rather heavy handed

Reply:
Maybe we're getting terms mixed up? I'm thinking of when the go pipeline is run, the molecules in the system get merged [here](https://github.com/marrink-lab/vermouth-martinize/blob/53c991df953d1e2f2b9763f5c165eab10c32f217/vermouth/rcsu/go_pipeline.py#L40) because that's how the Go model has to work. AFAICT when that happens at the moment that means _all_ molecule names go from `molecule_{0..n}` -> `molecule_0`. 

So, when you then take another protein you want in some complex system, you end up with `go_atomnames.itp` for multiple proteins, with associated `go_nbparams.itp`. However, because in each individual system the atomtypes and nonbonded parameters are notated the same, you end up with conflicting definitions. So if we allow users to define their own prefix for the name, then they can merge systems externally fine

Reply:
> that's how the Go model has to work

I think this is the bit that strikes me, because I don't see this. If I have a pdb with multiple proteins, not in contact, I don't see why they should all get merged. On the other hand, for hetero-dimers the merging does make sense.

In general I may have a different view of how people use martinize2. My idea is that you first produce your atomistic coordinates, containing all your proteins, and then you run the whole system through martinize2.

Reply:
ok, I think that explains the confusion! Everyone I've spoken to who makes these massive martini systems first makes their cg models to validate them individually, then combines them in their large system after.